pkg/urbit/vere/io/http.c

```diff
@@ -677,6 +677,8 @@ _http_start_respond(u3_hreq* req_u,
     if ( 0 == strncmp(hed_u->nam_c, "content-length", 14) ) {
       has_len_i = 1;
     }
+    // ignore connection specific headers for HTTP2
+    else if ( 0x200 <= rec_u->version && 0 == strncmp(hed_u->nam_c, "connection", 10) ) {}
```

Comment:
We should downcase all headers for http2 responses. I'd switch to something like this:

```
// ignore connection specific headers for HTTP2
//
if ( 0x200 <= rec_u->version ) {
  h2o_strtolower(hed_u->nam_c, hed_u->nam_w);
  
  if ( 0 == strncmp(hed_u->nam_c, "connection", 10) ) {
    continue;
  }
}

if ( 0 == strncmp(hed_u->nam_c, "content-length", 14) ) {
  has_len_i = 1;
}
```

Reply:
We might be able to get away with this for HTTP1 as well, but I've seen comments to the effect that title-case is maximally compatible, at least for key headers ...